### PR TITLE
Add contact

### PR DESCRIPTION
**Pull request by me-stevens**
I would like to know if this PR reads too much like java. I followed this [Ruby style guide](https://github.com/bbatsov/ruby-style-guide) and skimmed through some random 8th Light Ruby repos.

### Does:
* Ask the user for a contact's fields in the command line.
* Save each contact to an array.
* Save contacts as hashes.
* Prints the contact and a list of all at the end.
* Ask if the user wants to add another contact.
* Small but relevant commits, small steps

### Doesn't:
* Save contacts in a file (yet).
* Check user inputs.
* Use any Ruby version manager.
* Is not clean yet.
* Still no design, everything in the main class, `Conman`. The ui was asking to be extracted, though.

@jsuchy, @christophgockel

### CODE REVIEW

File: spec/ui_spec.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
+require 'console'
+require 'ui'
+
+describe UI do
+
+  let(:input)  {StringIO.new}
+  let(:output) {StringIO.new}
+  let(:ui)     {UI.new(Console.new(input, output))}
+
+  it "reads an answer" do
```
Comment (christophgockel) [inline spec/ui_spec.rb]: The name of this test tells me that something regarding reading an answer will be tested. But the test verifies two different responsibilities of `ask_for_another`.
One is that it displays something to the UI, and the other one is that it returns a value that a user enters via stdin.
(This is true for the majority of tests in here)

File: spec/conman_spec.rb [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  it "prints all contacts after finished" do
+    input.string << "y\n"
+    input.string << "name\naddress\n123456\nemail@mail.com\nnotes\n"
+    input.string << "n\n"
+    conman.run
+    expect(output.string).to include(UI::HEADER)
+  end
+
+  private
```
Comment (christophgockel) [inline spec/conman_spec.rb]: No need for a `private` section in a test class.
Helper methods are usually just added before or after the `it` blocks.

File: spec/conman_spec.rb [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    expect_field(:email, "email@mail.com")
+  end
+
+  it "saves the notes introduced by the user" do
+    conman.add_contact
+    expect_field(:notes, "notes")
+  end
+
+  it "adds two contacts through the main loop" do
+    input.string << "y\n"
```
Comment (christophgockel) [inline spec/conman_spec.rb]: This ties the high-level class `Conman` to the very concrete detail of string input.
Can you think of a better way to abstract away from these details?

File: lib/conman.rb [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+class Conman
+
+  def initialize(ui)
+    @ui           = ui
+    @contact_list = []
+  end
+
+  def run
+    answer = "y"
```
Comment (christophgockel) [inline lib/conman.rb]: This string is duplicated knowledge here.
Should it be the responsibility of `Conman` to interpret what a `y` means that a user entered?

**Comment by me-stevens** [comment]
@christophgockel your corrections were added :+1:

**Pull request merged**